pom.xml

```diff
@@ -17,7 +17,6 @@
 		<maven.compiler.target>11</maven.compiler.target>
 		<maven.compiler.source>11</maven.compiler.source>
 		<rxjava.version>2.2.16</rxjava.version>
-		<lombok.version>1.18.10</lombok.version>
```

Comment:
we are using lombok, Can we check if any unused dependency exists, if not we can close the issue.